docs/.gitignore

```diff
@@ -0,0 +1,11 @@
+/*env*/
```

Comment:
What is the benefit to add this only to this subdirectory? I would like to see (one day) usage of templates for files like this, so single `.gitignore`  sound better to me.

Reply:
I think that now I understand that all of these files are coming from some template.

Reply:
Yes, the idea is to keep as many files from the upstream repository intact as possible, so in future we can easily update it accordingly.